configuration/items.md

```diff
@@ -649,7 +649,8 @@ This parameter allows to post an update or command to an item after a period of
 
 The expiration timer is started or restarted every time an item receives an update or a command *other than* the specified "expire" update/command.
 Any future expiring update or command is cancelled, if the item receives an update or command that matches the "expire" update/command.
-
+The expire parameter can also be combined with items that are connected to a channel.
```

Comment:
I'd say that this is the normal case and thus should not be specifically mentioned.

Reply:
Maybe it´s not the best way to describe it.
I think it´s worth to mention this possible combination of channels and parameters.
Could it be an idea to add a general **Parameters** description above **Parameter `autoupdate`** and **Parameter `expire`**?
Like the general description of **Tags**.

Reply:
Yes, I guess it should go somewhere to the [binding configuration](https://next.openhab.org/docs/configuration/items.html#binding-configuration) section - after all, the `channel` and `autoupdate` and `expire` parameters are provided all the same way. Explaining this and thus introducing the additional parameters in a general Parameters section above `autoupdate` definitely makes sense!